tests/Model/CollectionInfoTest.php

```diff
@@ -16,7 +16,7 @@ public function testGetName()
     public function testGetOptions()
     {
         $info = new CollectionInfo(['name' => 'foo']);
-        $this->assertSame([], $info->getOptions());
+        $this->assertEmpty($info->getOptions());
```

Comment:
This assertion is less strict than `assertSame()`, as [`IsEmpty::matches()`](https://github.com/sebastianbergmann/phpunit/blob/485e7f707692b81c360e713c280601ad20de5392/src/Framework/Constraint/IsEmpty.php#L27) may defer to Countable or PHP's `empty()` construct. The original assertion very specifically tests that we get an empty array, which is consistent with `CollectionInfo::getOptions()` documenting that it returns an array type at all times.